lib/Helper/PackageManager.ts

```diff
@@ -5,6 +5,9 @@ import * as path from 'path';
 import { promisify } from 'util';
 
 export function getPackageManagerChoice(): PackageManager | null {
```

Comment:
If we use it I would rename this function to something like `detectPackageManager` or `detectPackageManager`.